dm_control/locomotion/walkers/assets/dog_v2/dog_base.xml

```diff
@@ -3,23 +3,24 @@
 
   <option timestep="0.005" noslip_iterations="4"/>
 
-  <size njmax="900" nconmax="300" nkey="1"/>
+  <size njmax="900" nconmax="900" nkey="1"/>
```

Comment:
Don't use this, use `memory`

https://mujoco.readthedocs.io/en/latest/modeling.html#memory-allocation